docs/docs.scss

```diff
@@ -281,8 +281,8 @@ body {
   .column {
     padding: $grid-gutter;
     font-family: $mono-font;
-    background-color: lighten($brand-green, 40%);
-    border: 1px solid lighten($brand-green, 30%);
+    background-color: lighten($green, 40%);
+    border: 1px solid lighten($green, 30%);
```

Comment:
Ah yeah, that's fair. Sorry didn't notice you were just changing the existing variable name.